tests/appsec/test_automated_login_events.py

```diff
@@ -246,11 +246,16 @@ def test_login_wrong_password_failure(self):
                     # Currently in nodejs there is no way to check if the user exists upon authentication failure so
                     # this assertion is disabled for this library.
                     assert meta["appsec.events.users.login.failure.usr.exists"] == "true"
+                    assert meta["appsec.events.users.login.failure.usr.id"] == "social-security-id"
+                    assert meta["appsec.events.users.login.failure.email"] == "testuser@ddog.com"
+                    assert meta["appsec.events.users.login.failure.username"] == "test"
+                else:
+                    assert meta["appsec.events.users.login.failure.usr.id"] == "test"
```

Comment:
not sure why in nodejs here, we dont retrieve the real user id, but the username? @hoolioh

Reply:
That's a peculiarity of the implementation because we the user info is not available in the implementation despite it exists so we're reporting the login as the id.

Reply:
as discussed: peculiarity from the SDK implementation where the examples for user login failure were reporting the login/username as 'usr.id' keeping the behavior as is in auto instrumentation. Hence, upon failure,  the 'login' is used and turned to an 'id' field.